kafka/clusters/prod-kafka-cluster.yaml

```diff
@@ -1,12 +1,18 @@
-apiVersion: kafka.strimzi.io/v1beta1
```

Comment:
@NoahNsimbe 
A quick one, could we have these k8s deployment files under the k8s directory.....just to maintain the original flow of things? @danielogen

Reply:
I guess this comment was intended to the corresponding PR which is a work in progress.......let me migrate it there shortly......I am of the view that we do not revert but rather concentrate on using the other corresponding PR to bring in new changes accordingly. Thanks.

Reply:
thanks @Baalmart , the requested changes have been worked on in #462